fury/actor.py

```diff
@@ -188,6 +189,85 @@ def copy(self):
     return image_actor
 
 
+def surface(vertices, faces=None, smooth=None):
+    temp1 = np.amax(vertices, axis=0)
+    temp2 = np.amin(vertices, axis=0)
+    size = list()
+    for i in range(len(temp1)):
+        if abs(temp1[i]) > abs(temp2[i]):
+            size.append(abs(temp1[i]))
+        else:
+            size.append(abs(temp2[i]))
+
+    colors = vtk.vtkUnsignedCharArray()
+    colors.SetNumberOfComponents(3)
+    points = vtk.vtkPoints()
+    triangles = vtk.vtkCellArray()
+    if faces is None:
+        xy = list()
+        for coordinate in vertices:
+            xy.append([coordinate[0], coordinate[1]])
+        tri = Delaunay(xy)
+        faces = tri.simplices
+
+    count = 0
+    for face in faces:
+        p_1 = vertices[face[0]]
+        p_2 = vertices[face[1]]
+        p_3 = vertices[face[2]]
+
+        points.InsertNextPoint(p_1[0], p_1[1], p_1[2])
```

Comment:
These loops can be avoided. Look here https://github.com/fury-gl/fury/blob/master/fury/utils.py#L351